app/business_model/models.py

```diff
@@ -78,3 +78,14 @@ class BMAnswer(models.Model):
         BusinessModel, on_delete=models.CASCADE, null=True, blank=False, related_name="user_answers"
     )
     score = models.FloatField(null=True, verbose_name="Score")
+
+
+class EquityData(models.Model):
+    scenario = models.ForeignKey(Scenario, on_delete=models.CASCADE, null=True, blank=True)
+    debt_start = models.IntegerField()
+    grant_share = models.FloatField()
```

Comment:
This is a number between 0 and 1, look at the validators for `soc_max` in projects models